go-app-ussd_optout_rapidpro.js

```diff
@@ -334,32 +344,33 @@ go.app = function() {
                     if(_.includes(["miscarriage", "stillborn", "babyloss"], choice.value)) {
                         return "state_loss_optout";
                     } else {
-                        return "state_nonloss_optout";
+                        return "state_trigger_rapidpro_flow";
                     }
                 }
             });
         });
 
         self.add("state_trigger_rapidpro_flow", function(name, opts) {
             var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
+            var answers = self.im.user.answers;
+
+            var forget = answers.forget_optout;
+            var loss = answers.state_loss_optout === "yes" ? "TRUE" : "FALSE";
+
             return self.rapidpro.start_flow(self.im.config.flow_uuid, null, "whatsapp:" + _.trim(msisdn, "+"), {
-                babyloss_subscription:
-                    self.im.user.get_answer("state_loss_optout") === "yes" ? "TRUE" : "FALSE",
-                delete_info_for_babyloss:
-                    self.im.user.get_answer("state_delete_info_for_loss") === "yes" ? "TRUE" : "FALSE",
-                delete_info_consent:
-                        self.im.user.get_answer("state_delete_research_info") === "yes" ? "TRUE" : "FALSE",
+                babyloss_subscription: loss,
+                delete_info_for_babyloss: forget ? "TRUE" : "FALSE",
+                delete_info_consent: forget ? "TRUE" : "FALSE",
```

Comment:
Having `forget` be boolean and `loss` be text can be a bit confusing. If we make them the same (either both TRUE/FALSE or both boolean), then I think that removes that confusion, and can probably avoid making a mistake with the type.

Reply:
good point, thanks